Persistence/scheduled task creation.txt

```diff
@@ -0,0 +1,6 @@
+//Original Sigma Rule: https://github.com/Neo23x0/sigma/blob/master/rules/windows/process_creation/win_susp_schtask_creation.yml
+//Questions via Twitter: @janvonkirchheim 
+
+ProcessCreationEvents 
+| where FolderPath endswith "\\schtasks.exe" and ProcessCommandLine has " /create " and AccountName != "system"
+| where EventTime > ago(30d)
```

Comment:
The default time filter in Advanced Hunting page is 30 days. In this repo we usually reduce that to 1d or 7d if it's rare - and the SecOps running this will change according to the results - iteratively.